src/main/java/net/glowstone/net/GlowDatagramServer.java

```diff
@@ -18,6 +18,12 @@
     protected final EventLoopGroup group;
     protected final Bootstrap bootstrap;
 
+    /**
+     * Creates an instance for the specified server.
+     *
+     * @param server the associated GlowServer
+     * @param latch TODO: document this parameter
```

Comment:
The countdown latch used during server startup to wait for network server binding.